assets/js/pages/ClusterDetails/HanaClusterDetails.jsx

```diff
@@ -51,10 +52,12 @@ function HanaClusterDetails({
   navigate = () => {},
 }) {
   const enrichedNodes = enrichNodes(details?.nodes, hosts);
-  const enrichedSapSystem = {
-    sid,
-    ...sapSystems.find(({ sid: currentSid }) => currentSid === sid),
-  };
+  const sortedSidList = [sid, ...additionalSids].sort();
```

Comment:
We don't want to sort it. We want to have first the main sid and the QAS later on.
Just remove the `sort` function and rename the variable to `clusteredSids` maybe